code/frontend/src/main.js

```diff
@@ -0,0 +1,8 @@
+import './app.css'
```

Comment:
I think in the next sprint we should switch to TypeScript.

Reply:
On Sunday @rabbit-zero, @jenswaechtler and me spoke about whether we want to use JavaScript or TypeScript, we decided on JavaScript. If you want to we can talk about it tomorrow.